ts/tests/helpers/skeleton-app.ts

```diff
@@ -44,7 +44,7 @@ export default class SkeletonApp {
     ));
   }
 
-  updatePackageJSON(callback: (arg: any) => string) {
+  updatePackageJSON(callback: (arg: any) => any) {
```

Comment:
😭

Reply:
Really this is `(arg: PackageJSON) => PackageJSON | void` — `string` was never right 🙃